### ISSUE TITLE
Installing from wheel, no pyRipser

### ISSUE THREAD

**Issue by mjcatanz** [scikit-tda/ripser.py#73]
I was unable to install ripser using `pip install ripser` so I installed ripser directly from the whl file at https://pypi.org/project/ripser/#files. Now `from ripser import Rips` gives 

```
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-2-777271b17a02> in <module>
      1 import numpy as np
----> 2 from ripser import Rips

~\.conda\envs\datasci_py37\lib\site-packages\ripser\__init__.py in <module>
----> 1 from .ripser import *
      2 
      3 from ._version import __version__

~\.conda\envs\datasci_py37\lib\site-packages\ripser\ripser.py in <module>
     36 import persim
     37 
---> 38 from pyRipser import doRipsFiltrationDM as DRFDM
     39 from pyRipser import doRipsFiltrationDMSparse as DRFDMSparse
     40 

ModuleNotFoundError: No module named 'pyRipser'
```
(I'm sure this is not the intended way of installing, but I ran into the python error about Microsoft VS 14.0, and then "error: command 'C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\cl.exe' failed with exit status 2". As I understand it, these are python errors, unrelated to the minGW installation.)

**Comment by sauln** [comment]
There were no errors when you installed it from the wheel?  It looks like you've installed it using the VS compiler instead of minGW.  Do you have minGW set up?

**Comment by mjcatanz** [comment]
No errors from wheel... And I had minGW installed when `pip install ripser` bombed.

**Comment by sauln** [comment]
Could you try following these steps for the install. They work on Appveyor with conda:

https://github.com/scikit-tda/ripser.py/blob/6ae34fc9e9589c2bf3758a948355c73d1f29c9a5/appveyor.yml#L26-L39

Some of the steps might not be necessary, but I think the lines

```
  - conda update -q conda 
  - conda install libpython m2w64-toolchain cython
  - echo [build] > %CONDA_PREFIX%\Lib\distutils\distutils.cfg
  - echo compiler = mingw32 >> %CONDA_PREFIX%\Lib\distutils\distutils.cfg
```

will be necessary before installation. The `%CONDA_PREFIX%` should be changed to where ever your conda is installed.

**Comment by mjcatanz** [comment]
So that totally worked! Could you shed some light on why that succeeded but `pip install ripser` or the wheel file didn't?

**Comment by sauln** [comment]
That's great! I'm glad it worked. I'm still trying to figure out how conda+windows+mingw play together, but I think what's happening here is you are explicitly telling conda to use mingw compiler. 

For sake of reproducability, could you share the exact commands you made to get the install to work? This process needs much better documentation.

**Comment by mjcatanz** [comment]
After working on the install on my own and a couple other windows machines, I think the issue is exactly as you say--forcing conda to use mingw to compile as opposed to whatever default setting is. The errors I came across installing ripser before this particular one I asked about were related to general C++/Cython dependent issues, like pip installing [steem](https://stackoverflow.com/questions/44951456/pip-error-microsoft-visual-c-14-0-is-required), [misaka](https://stackoverflow.com/questions/48541801/microsoft-visual-c-14-0-is-required-get-it-with-microsoft-visual-c-build-t), or related to missing/invalid [cl.exe](https://stackoverflow.com/questions/41724445/python-pip-on-windows-command-cl-exe-failed).

In terms of reproducibility, I did exactly the steps you mentioned above in the larger scroll box, not just the four lines you highlighted, except the first--I didn't modify the path. The same commands also worked in my standard environment, so I didn't have to create a new test-environment.
